rbac/management/group/definer.py

```diff
@@ -105,7 +105,7 @@ def clone_default_group_in_public_schema(group, tenant) -> Optional[Group]:
         group_uuid = uuid4()
 
     public_tenant = Tenant.objects.get(tenant_name="public")
-    tenant_default_policy = group.policies.get(system=True)
```

Comment:
For the policies, the situation is different, we used system True to indicate the policies can not be updated, but the tenant is in each tenant. Maybe we can just ignore the system=True for policies. You might need to do some research if we have any system not true policies